src/main/java/org/jenkinsci/plugins/fodupload/Utils.java

```diff
@@ -182,4 +187,31 @@ public static String retrieveSecretDecryptedValue(String id) {
     public static Boolean isUnauthorizedResponse(Response response) {
         return response.code() == HttpStatus.SC_FORBIDDEN || response.code() == HttpStatus.SC_UNAUTHORIZED;
     }
+
+    public static String createApplicationResponseViewModel(List<ApplicationApiResponse> applicationResponseList) {
+        Gson gson = new Gson();
+//        HashMap<Integer, String> viewObject = new HashMap<Integer, String>();
+//        for (ApplicationApiResponse applicationResponse : applicationResponseList) {
+//            viewObject.put(applicationResponse.getApplicationId(), applicationResponse.getApplicationName());
+//        }
+        return gson.toJson(applicationResponseList);
```

Comment:
This seems to be a bad practice because it relies on how list elements' "toString()" method is implemented. Besides, for the typed collections gson recommends using the typed version of toJson method, see https://www.javadoc.io/doc/com.google.code.gson/gson/2.8.5/com/google/gson/Gson.html#toJson-java.lang.Object-java.lang.reflect.Type-